virttest/qemu_vm.py

```diff
@@ -4,6 +4,7 @@
 :copyright: 2008-2009, 2014 Red Hat Inc.
 """
 
+from __future__ import division
```

Comment:
Any other update for this file?

Reply:
Do you mean do I need to update the codes in this file?

Reply:
I have viewed every line (unless I accidentally missed some ) that uses / operator, if it produces same results for both Python 2 and 3. I **did not** change the original code, just added `from __future__ import division` to make sure the / operator to behave the
same in Python 2 as it does in Python 3

I changed some  / to //, that is I believe it was intended to get a integer out of the division operation. If i did not update it, it would get a float number in Python 3.

Reply:
@hereischen If the codes don't need to be updated, the old division works well in Python 2. And the module is not necessary to import

Reply:
How can you make sure they are same without running it?

Reply:
I mean the result is integer in Python 2 before you import the module, and the avocado works well now(with integer). And you are sure it will also work well when it get a float. So if you think it is necessary to make sure the behave the same, // operator is better because it keep the original result.